README.md

```diff
@@ -15,7 +15,13 @@ This library is the canonical location of the Open Cluster Management API defini
 
 ## Consumers
 
-Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. For example, see [registration-operator](https://github.com/open-cluster-management-io/registration-operator) for how the `api` is being consumed.
+Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. 
+
+* [registration](https://github.com/open-cluster-management-io/registration): implements `ManagedCluster`, `ManagedClusterSet`,`ClusterClaim` for cluster registration and lifecycling.
+* [work](https://github.com/open-cluster-management-io/work): implements `Manifestwork` for kubernetes resource distribution 
```

Comment:
for native kubernetes resource distribution to multiple clusters.

Reply:
Done